scripts/download_image.py

```diff
@@ -47,13 +48,25 @@ def __init__(self, total: int, report_function: callable, *args, **kwargs):
                                                  **kwargs)
         self._validate_report_function()
 
-    def report(self, message, multiline=True, *args, **kwargs):
-        if not multiline:
+    def report(self, message: str, next_line: bool = True, *args, **kwargs):
+        """
+        Prints requested message to the console.
+        :param next_line: whatever reporting should skip a line or not.
+        :type message: str message to print to the console.
+        :return:
+        """
+        if not next_line:
```

Comment:
This is _progress_reporter specific code?

This change increase test costs, another reporter also may take the message with next_line.

Reply:
Added some logic to all reporters and removed string appending